test/unit/controller/service_switch_controller_test.js

```diff
@@ -27,29 +27,27 @@ describe('service switch controller: list of accounts', function () {
     renderSpy = sinon.spy()
   })
 
-  it('should render a list of services when user has multiple services', function (done) {
+  it.only('should render a list of services when user has multiple services', function (done) {
```

Comment:
naughty

Reply:
Whoops. I should have had [WIP] on this PR the whole time as its being actively worked on.